tests/e2e/test_e2e.py

```diff
@@ -18,166 +16,152 @@
-            executors.JavaExecutor,
-            utils.train_model_classification_binary,
-            marks=[JAVA, CLASSIFICATION],
-    ),
-    pytest.param(
-            ensemble.RandomForestClassifier(n_estimators=10,
-                                            random_state=RANDOM_SEED),
-            executors.JavaExecutor,
-            utils.train_model_classification,
-            marks=[JAVA, CLASSIFICATION],
-    ),
-    pytest.param(
-            linear_model.LinearRegression(),
-            executors.PythonExecutor,
-            utils.train_model_regression,
-            marks=[PYTHON, REGRESSION],
-    ),
-    pytest.param(
-            tree.DecisionTreeRegressor(random_state=RANDOM_SEED),
-            executors.PythonExecutor,
-            utils.train_model_regression,
-            marks=[PYTHON, REGRESSION],
-    ),
-    pytest.param(
-            ensemble.RandomForestRegressor(n_estimators=10,
-                                           random_state=RANDOM_SEED),
-            executors.PythonExecutor,
-            utils.train_model_regression,
-            marks=[PYTHON, REGRESSION],
-    ),
-    pytest.param(
-            linear_model.LogisticRegression(),
-            executors.PythonExecutor,
-            utils.train_model_classification,
-            marks=[PYTHON, CLASSIFICATION],
-    ),
-    pytest.param(
-            linear_model.LogisticRegression(),
-            executors.PythonExecutor,
-            utils.train_model_classification_binary,
-            marks=[PYTHON, CLASSIFICATION],
-    ),
-    pytest.param(
-            tree.DecisionTreeClassifier(random_state=RANDOM_SEED),
-            executors.PythonExecutor,
-            utils.train_model_classification,
-            marks=[PYTHON, CLASSIFICATION],
-    ),
-    pytest.param(
-            tree.DecisionTreeClassifier(random_state=RANDOM_SEED),
-            executors.PythonExecutor,
-            utils.train_model_classification_binary,
-            marks=[PYTHON, CLASSIFICATION],
-    ),
-    pytest.param(
-            ensemble.RandomForestClassifier(n_estimators=10,
-                                            random_state=RANDOM_SEED),
-            executors.PythonExecutor,
-            utils.train_model_classification_binary,
-            marks=[PYTHON, CLASSIFICATION],
-    ),
-    pytest.param(
-            ensemble.RandomForestClassifier(n_estimators=10,
-                                            random_state=RANDOM_SEED),
-            executors.PythonExecutor,
-            utils.train_model_classification,
-            marks=[PYTHON, CLASSIFICATION],
-    ),
+
+RANDOM_SEED = 1234
+FOREST_PARAMS = dict(n_estimators=10, random_state=RANDOM_SEED)
+
+
+# Reusable models for e2e tests. They will be cloned on each execution.
+linear_regressor = linear_model.LinearRegression()
+logistic_regressor = linear_model.LogisticRegression()
+decision_tree_regressor = tree.DecisionTreeRegressor()
+decision_tree_classifier = tree.DecisionTreeClassifier()
```

Comment:
`random_state` got lost

Reply:
Looks like it was fixed in #53 :) thanks!